src/index.js

```diff
@@ -14,7 +14,21 @@ export default {
 	async fetch(request, env, ctx) {
 		const url = new URL(request.url);
 
-		const apiConfig = await env.CONFIG.get("api-config.json");
+		let apiConfig;
+		try {
+			apiConfig = await env.CONFIG.get("api-config.json");
+			if (!apiConfig) {
+				throw new Error("KV value not found");
```

Comment:
This throw and catch feels redundant. Instead of nested try-catch blocks, in if block, you can check the local file.

Reply:
I'm working on that but failed last night. I'll check it again today.